app/containers/GameList/GameList.js

```diff
@@ -3,23 +3,40 @@ import PropTypes from 'prop-types';
 import { GridList, GridTile } from 'material-ui/GridList';
 import IconButton from 'material-ui/IconButton';
 import ThumbUp from 'material-ui/svg-icons/action/thumb-up';
-import { FlatButton } from 'material-ui';
+import { AppBar, FlatButton } from 'material-ui';
 import TokaidoImage from 'images/tokaido.jpg';
 
+const buttonStyle = {
```

Comment:
We must, later, migrate fully to styled components and remove styling from components (specially containers)